components/admin_console/admin_definition.jsx

```diff
@@ -182,11 +182,11 @@ export const it = {
     enterpriseReady: (config, state, license, enterpriseReady) => enterpriseReady,
     licensed: (config, state, license) => license.IsLicensed === 'true',
     licensedForFeature: (feature) => (config, state, license) => license.IsLicensed && license[feature] === 'true',
-    isPaidTier: (config, state, license, enterpriseReady, consoleAccess, cloud) => {
+    hidePaymentInfo: (config, state, license, enterpriseReady, consoleAccess, cloud) => {
         if (!cloud?.subscription) {
             return false;
         }
```

Comment:
If now this is to `hide` then if there is no cloud subscription then we do want to hide so it shouldn't return `false` here anymore.

Wondering if we just remove this check and add the null checks below?

Reply:
thanks, good catch! fixing it now.